.github/workflows/run-ide-e2e.yaml

```diff
@@ -0,0 +1,136 @@
+name: Run IDE e2e on release
+on:
+  workflow_call:
+
+# this worflow copies the exact IDE e2e and runs it on the release branches
+jobs:
+  intellij-e2e:
```

Comment:
Do you need to copy the entire workflow? Can't you trigger the original workflow with something like this? https://github.com/marketplace/actions/trigger-external-workflow

Reply:
- since we want to block in case something fails, i couldnt find a way to do that with this, i also looked at other actions that use repository_dispatch
- we want to test with mirrord binary built on the tag created in the pr, and add it to the path

what we can do is, move the reusable part of the workflow to `mirrord-intellij` and use that here as well